fenix/app/src/main/java/org/mozilla/fenix/messaging/MessageNotificationWorker.kt

```diff
@@ -170,28 +170,27 @@ class MessageNotificationWorker(
  * When a [Message] [Notification] is dismissed by the user record telemetry data and update the
  * [Message.metadata].
  *
- * This [Service] is only intended to be used by the [MessageNotificationWorker.createOnDismissPendingIntent] function.
+ * This Service is only intended to be used by the [MessageNotificationWorker.createOnDismissPendingIntent] function.
  */
-class NotificationDismissedService : Service() {
-
-    /**
-     * This service cannot be bound to.
-     */
-    override fun onBind(intent: Intent?): IBinder? = null
```

Comment:
afaict this behaviour is the same as the default of `LifecycleService` `onBind` fun - difference here is the additional `dispatcher` call for handling the lifecycle